guide/interactions/slash-commands.md

```diff
@@ -110,6 +113,28 @@ const data = new SlashCommandBuilder()
 			.setDescription('The input to echo back')
 			.setRequired(true));
 ```
+:::
+
+::: code-group-item JSON
+
+```js
+const data = {
+	name: 'echo',
+	description: 'Replies with your input!',
+	options: [
+		{
+			name: 'input',
+			description: 'The input to echo back',
+			type: 3,
```

Comment:
I recommended the usage of enums, is there a reason why those weren't used? They can be imported from discord-api-types
If you use rest, you need them for the routes, and rest uses it, builders uses it, so does discord.js
I don't think there's really a noticeable downside to using enums